inductiva/tasks/task.py

```diff
@@ -179,110 +178,13 @@ def kill(self) -> None:
         """
         self._api.kill_task(path_params=self._get_path_params())
 
-    def _get_output_config(self, all_files: bool = False):
-        """Get configuration of the output with the task method name.
-
-        Args:
-            all_files: Whether to download all the files in the output.
-
-        Returns:
-            output_class, filenames - If the task method name is not of
-                a scenario, this method returns None, None and all
-                simulation are downloaded. Otherwise, and if available,
-                it returns the output_class of that
-                respective scenario and the filenames of that simulation.
-                If all_files is False, then the filenames are the default
-                files set for that scenario.
-        """
-
-        # Fetch the first part of the method_name (e.g., "wind_tunnel")
-        method_name = self.get_scenario_name()
-
-        # Set the default files for the output class
-        # For some scenarios, there are None default files
-        filenames = None
-        output_class = None
-
-        if method_name in output_consts.OUTPUT_CONSTS:
-            output_class = output_consts.OUTPUT_CONSTS[method_name][
-                "output_class"]
-            if not all_files:
-                filenames = output_consts.OUTPUT_CONSTS[method_name][
-                    "default_files"]
-
-        return output_class, filenames
-
-    def get_scenario_name(self) -> Optional[str]:
-        name = self.get_info()["method_name"].split(".")[0]
-        if name in output_consts.OUTPUT_CONSTS:
-            return name
-
-        return None
-
     def get_simulator_name(self) -> str:
         # e.g. retrieve openfoam from fvm.openfoam.run_simulation
         return self.get_info()["method_name"].split(".")[1]
 
     def get_storage_path(self) -> str:
         return self.get_info()["storage_path"]
 
-    def get_output(
```

Comment:
Why were these methods removed? Where were they moved to?

Reply:
They were removed because they were attached to the scenarios existence. Their sole purpose was to fetch the output class of the scenarios.